### PR TITLE
Parse query strings

### PR DESCRIPTION
**Pull request by ericdrosado**

### CODE REVIEW

**Review by cdemyanovich — COMMENTED** [review]
When I request `/` after `/hello?noun=Craig`, I still see "Hello Craig" in the response instead of "Hello World".

File: Sources/Server/Response.swift [hunk 1]
```diff
@@ -1,41 +1,60 @@
...
-            return header
-        } 
-    }
-    
+   }
+
+   private func parsePathForQueries(path: String) -> String {
+        let parsedQueries = path.split(separator: "?") 
+        let query = parsedQueries[1].split(separator: "=") 
+        self.queries.updateValue(String(query[1]), forKey: String(query[0]))
```
Comment (tommcgee) [inline Sources/Server/Response.swift]: why is `self` necessary here but not line 22?

**Review by tommcgee — COMMENTED** [review]
Outside of Craig's comment and a few inconsistencies, looks good to me.

File: Sources/Server/Response.swift [hunk 2]
```diff
@@ -21,6 +21,7 @@ public class Response {
        if (paths.contains(path)) {
            body = prepareBody(body: "Hello \(queries["noun"]!)", method: method)
            header = buildHeader(statusCode: status200, contentLength: body.utf8.count) 
+           queries.updateValue("World", forKey: "noun")
```
Comment (cdemyanovich) [inline Sources/Server/Response.swift]: How did the test you added necessitate this change?

**Review by cdemyanovich — APPROVED** [review]

**Pull request merged**